net/netlink/Make.defs

```diff
@@ -26,7 +26,7 @@ SOCK_CSRCS += netlink_sockif.c
 NET_CSRCS += netlink_conn.c netlink_notifier.c
 
 ifeq ($(CONFIG_NETLINK_ROUTE),y)
-NET_CSRCS += netlink_route.c
+NET_CSRCS += netlink_route.c nlattr.c
```

Comment:
Should we name file `netlink_attr.c` to be consistent with other files in the netlink folder?